projects/igniteui-angular/src/lib/services/csv/csv-exporter-options.ts

```diff
@@ -29,18 +32,30 @@ export class IgxCsvExporterOptions extends IgxExporterOptionsBase {
         return extension;
     }
 
+    /**
+     * Gets the value delimiter which will be used for the exporing operation.
```

Comment:
Typo - "exporing" instead of "exporting".

Reply:
Fixed